stdlib/2and3/argparse.pyi

```diff
@@ -79,8 +79,9 @@ class ArgumentParser:
     def print_help(self, file: Optional[IO[str]] = ...) -> None: ...
     def format_usage(self) -> str: ...
     def format_help(self) -> str: ...
-    def parse_known_args(self, args: Optional[str] = ...,
-                         namespace: Optional[Namespace] = ...) -> Tuple[Namespace, List[str]]: ...
+    def parse_known_args(self, args: Optional[Sequence[str]] = ...,
+                         namespace: Optional[Namespace] = ...) \
```

Comment:
Can you avoid using \ for line continuations? Someone else is doing this (apparently the pytype parser doesn't like them yet).